pokemon-checker/src/components/PokemonDisplay/DisplayFunctionalComponents/PokemonImageDisplay.tsx

```diff
@@ -18,10 +23,23 @@ export const PokemonImage = ({
   return (
     <div>
       {
-        <img id="pokemonDisplay" src={defaultFront} alt={altImageName}></img>
+        <
+          img 
+          id="pokemonDisplay" 
```

Comment:
Just realized that because we may at some point render multiple pokemon, we should use a `class` instead of an `id`. According to the docs, [IDs can only be used once per page](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Type_Class_and_ID_Selectors#id_selectors).